app/apps/cases/views.py

```diff
@@ -78,6 +81,20 @@ def unplanned(self, request):
 
         return JsonResponse({"cases": cases})
 
+    @extend_schema(
+        description="Get all visits connected to this case",
+        responses={200: VisitSerializer(many=True)},
+    )
+    @action(detail=True, methods=["get"], name="get_log_timeline")
+    def get_log_timeline(self, request, pk):
+        """
+        Get list of all visits to the building of this case
```

Comment:
Ik zou dit gewoon 'Get all visits connected to this case' maken (net als je schema description). Er kunnen meerdere zaken op één adres zijn, en een visit is echt alleen gekoppeld aan een case op dit moment (dus 'buidling' mag weg)

Is dit voor de zaak detail pagina?

Reply:
Is iid voorwerk voor zaak detail pagina. dacht zet het er maal al snel bij en dan zien ze dat er meerdere "descriptions" zijn en dat we iets moeten aanpassen aan design/frontend voor timeline op zaak detail page